validphys2/examples/true_pos_neg_closuretests.yaml

```diff
@@ -0,0 +1,114 @@
+meta:
+  title: Inconsistency classification probabilities
+  author: Lazy Person
+  keywords: [nsigma, chi2, multiclosure test, inconsistent]
+
```

Comment:
Add perhaps a comment saying that the commented dataset are those to reproduce the paper.